apps/OboeTester/app/src/main/res/values/strings.xml

```diff
@@ -100,6 +100,7 @@
         <item>PCM_I24</item>
         <item>PCM_I32</item>
         <item>IEC61937</item>
+        <item>MP3</item>
```

Comment:
What happens to other params like sample rates when this is set?

Reply:
They are not changed. The users must manually change the configuration. This is added in the doc and error toast.